### PR TITLE
fib: make FIB size dependent on existence of RPL

### PR DESCRIPTION
**Pull request by OlegHahm**
Also, enables to configure the FIB size from the applicatin

### CODE REVIEW

File: sys/include/net/gnrc/ipv6.h [hunk 1]
```diff
@@ -83,7 +83,13 @@ extern kernel_pid_t gnrc_ipv6_pid;
 /**
  * @brief   Maximum number of entries in the IPv6 FIB table.
  */
-#define GNRC_IPV6_FIB_TABLE_SIZE        (20)
+#ifndef GNRC_IPV6_FIB_TABLE_SIZE
+#   ifdef MODULE_GNRC_RPL
+#       define GNRC_IPV6_FIB_TABLE_SIZE     (20)
```
Comment (BytesGalore) [inline sys/include/net/gnrc/ipv6.h]: nice, have not tested yet but what if i set it up like:
```
GNRC_IPV6_FIB_TABLE_SIZE=0 make
```
?

Comment (OlegHahm) [inline sys/include/net/gnrc/ipv6.h]: Will probably fail because C99 doesn't allow zero-sized arrays

**Comment by cgundogan** [comment]
sensible. ACK from my side.

File: sys/include/net/gnrc/ipv6.h (hunk 1 above)
Comment (BytesGalore) [inline sys/include/net/gnrc/ipv6.h]: oh, my fault :eyeglasses: its from application not from command line

**Comment by cgundogan** [comment]
travis agrees - GO

**Pull request merged**